cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -2503,51 +2515,243 @@ def _run_api_v1_tasks_id_annotations(self, owner, assignee, annotator):
+                        "points": [1.0, 2.1, 51, 36.6],
+                        "type": "rectangle",
+                        "occluded": False,
+                        "outside": True
+                    }
+                ]
+            }]
+
+            rectangle_shapes_with_attrs = [{
+                "frame": 0,
+                "label_id": task["labels"][0]["id"],
+                "group": 0,
+                "attributes": [
+                    {
+                        "spec_id": task["labels"][0]["attributes"][0]["id"],
+                        "value": task["labels"][0]["attributes"][0]["values"][0]
+                    },
+                    {
+                        "spec_id": task["labels"][0]["attributes"][1]["id"],
+                        "value": task["labels"][0]["attributes"][1]["default_value"]
+                    }
+                ],
+                "points": [1.0, 2.1, 10.6, 53.22],
+                "type": "rectangle",
+                "occluded": False
+            }]
+
+            rectangle_shapes_wo_attrs = [{
+                "frame": 1,
+                "label_id": task["labels"][1]["id"],
+                "group": 0,
+                "attributes": [],
+                "points": [2.0, 2.1, 40, 50.7],
+                "type": "rectangle",
+                "occluded": False
+            }]
+
+            polygon_shapes_wo_attrs = [{
+                "frame": 1,
+                "label_id": task["labels"][1]["id"],
+                "group": 0,
+                "attributes": [],
+                "points": [2.0, 2.1, 100, 30.22, 40, 77, 1, 3],
+                "type": "polygon",
+                "occluded": False
+            }]
 
+            annotations = {
+                    "version": 0,
+                    "tags": [],
+                    "shapes": [],
+                    "tracks": [],
+                }
+            if annotation_format == "CVAT XML 1.1 for videos":
+                annotations["tracks"] = rectangle_tracks_with_attrs + rectangle_tracks_wo_attrs
+
+            elif annotation_format == "CVAT XML 1.1 for images":
+                annotations["shapes"] = rectangle_shapes_with_attrs + rectangle_shapes_wo_attrs
+
+            elif annotation_format == "PASCAL VOC ZIP 1.0" or \
+                 annotation_format == "YOLO ZIP 1.0" or \
+                 annotation_format == "TFRecord ZIP 1.0":
+                 annotations["shapes"] = rectangle_shapes_wo_attrs
+
+            elif annotation_format == "COCO JSON 1.0":
+                annotations["shapes"] = polygon_shapes_wo_attrs
+
+            elif annotation_format == "MASK ZIP 1.0":
+                annotations["shapes"] = rectangle_shapes_with_attrs + rectangle_shapes_wo_attrs + polygon_shapes_wo_attrs
+                annotations["tracks"] = rectangle_tracks_with_attrs + rectangle_tracks_wo_attrs
+
+            return annotations
+
+        for annotation_format in AnnotationFormat.objects.all():
```

Comment:
Can we use `server/formats` method from REST API? Can we add a UT for the method as well (check that `server/formats` return a list of expected formats)?

I'm not sure that the check anywhere that AnnotationFormat.objects.all() isn't empty.